apps/meteor/app/lib/server/methods/sendMessage.ts

```diff
@@ -140,6 +145,8 @@ Meteor.methods<ServerMethods>({
 			federation: Match.Maybe(Object),
 			groupable: Match.Maybe(Boolean),
 			sentByEmail: Match.Maybe(Boolean),
+			u: Match.Maybe(Object),
+			_updatedAt: Match.Maybe(Date),
```

Comment:
```suggestion
```
`_updatedAt` doesn't exist on unsent messages and `u` is filled in by the server to avoid impersonation. There's something wrong with the client if it's trying to pass either of those properties.

Reply:
removing it